submarine-workbench/workbench-server/src/main/java/org/apache/submarine/database/utils/HttpRequestUtil.java

```diff
@@ -0,0 +1,187 @@
+        "\t\"visibility\": \"PROJECT_VISIBILITY_PRIVATE\",\n" +
+        "\t\"permission\": \"PROJECT_PERMISSION_VIEW\",\n" +
+        "\t\"starNum\": 0,\n" +
+        "\t\"likeNum\": 0,\n" +
+        "\t\"messageNum\": 0,\n" +
+        "\t\"createBy\": \"userName\",\n" +
+        "\t\"updateBy\": \"userName\"\n" +
+        "}";
+    String addParameter = "{\n" +
+        "\"name\": \"test_project2\",\n" +
+        "\"visibility\": \"PROJECT_VISIBILITY_PRIVATE\",\n" +
+        "\"type\": \"PROJECT_TYPE_NOTEBOOK\",\n" +
+        "\"permission\": \"PROJECT_PERMISSION_VIEW\",\n" +
+        "\"tags\": null,\n" +
+        "\"starNum\": 0,\n" +
+        "\"likeNum\": 0,\n" +
+        "\"messageNum\": 0,\n" +
+        "\"teamName\": \"\",\n" +
+        "\"description\": \"test_project2\",\n" +
+        "\"userName\": \"test\",\n" +
+        "\"projectFilesList\": [],\n" +
+        "\"createBy\": \"test\",\n" +
+        "\"createTime\": \"2019-10-16 15:58:52\",\n" +
+        "\"updateBy\": \"test\",\n" +
+        "\"updateTime\": \"2019-10-16 15:58:52\"\n" +
+        "}";
+
+    String deleteParameter = "3ff46684a03011e5a63d00ff4d7a64c3";
+    try {
+      // String result =
+      // sendHttpRequest("http://localhost:8080/api/project/edit", null, editParameter.getBytes(), "PUT");
+      String result = sendHttpRequest("http://localhost:8080/api/project/add",
+          null, addParameter.getBytes(), "POST");
+      // String result = sendHttpRequest("http://localhost:8080/api/project/delete?id="
+      // + deleteParameter, null, null, "DELETE");
+      Map<String, String> map = new HashMap<>();
+      map.put("Authorization", "token 5e714c5f2798c329a5198aa2fd53c473dxxxxxxf");
+      // watching
+      // String result =
+      // sendHttpRequest("https://api.github.com/repos/zhulinhao306/test/subscribers", null);
+      // String result =
+      // sendHttpRequest("https://api.github.com/repos/zhulinhao306/test/subscribers", null, null,"GET");
+      // String result =
+      // sendHttpRequest("https://api.github.com/repos/zhulinhao306/test/subscription", map, null, "DELETE");
+      // String result =
+      // sendHttpRequest("https://api.github.com/repos/zhulinhao306/test/subscription", map, null, "PUT");
+
+      // starring
+      // String result =
+      // sendHttpRequest("https://api.github.com/repos/zhulinhao306/test/stargazers", null, null, "GET");
+      // String result =
+      // sendHttpRequest("https://api.github.com/user/starred/zhulinhao306/test", map, null, "DELETE");
+      // String result =
+      // sendHttpRequest("https://api.github.com/user/starred/zhulinhao306/test", map, null, "PUT");
+
+      // forks
+      // String result =
+      // sendHttpRequest("https://api.github.com/repos/zhulinhao306/test/forks", null, null, "GET");
+      // String result =
+      // sendHttpRequest("https://api.github.com/repos/zhulinhao306/test/forks", map, null, "POST");
+      System.out.println(result);
```

Comment:
Change `System.out.println(result);` to `LOG.info(result)`

Reply:
Done.